### DISCUSSION TITLE
Distribute template with children

### DISCUSSION THREAD

**Discussion by brianteeman** [joomla/joomla-cms#37358] [Q&A]
Is it possible (if so how) to build an installable template zip for a parent template with its children?

**Comment by dgrammatiko** [comment]
Yes, it should be possible. Here's *my* approach solving this:
- Have both the parent and child templates as zip files (eg a zip file with the contents of the templates/name folder and also the media/... files)
- Create a package with these zip files.
- Probably have a script file to apply the styles (overwrite or create new ones). This is easier than keeping the default XML values in sync with the DB values of each of the styles.

One note here: there is an assumption that the parent template is your **own** template and not trying to modify the **actual** Cassiopeia which should be avoided and requires a beefier package script.

**Reply by brianteeman** [reply]
Yes the parent is my own template.

I will have a try later on and see - thanks
